test/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ foreach(test ${tests})
   add_executable(${test} EXCLUDE_FROM_ALL ${test}.cc )
   set_target_properties(${test} PROPERTIES
     COMPILE_DEFINITIONS VALHALLA_SOURCE_DIR="${VALHALLA_SOURCE_DIR}/")
-  target_link_libraries(${test} valhalla gtest gmock)
+  target_link_libraries(${test} valhalla gtest gmock -lpthread -lm)
```

Comment:
if you enable the address sanitizer and you try to run the tests they have linker errors without the pthread flag. i didnt try it without libm so i should do that to see if i can remove it.